crates/abi/abi/HEVM.sol

```diff
@@ -82,7 +82,7 @@ expectRevert(bytes)
 expectRevert(bytes4)
 record()
 accesses(address)(bytes32[],bytes32[])
-skip(bool)
+skipTest(bool)
```

Comment:
should we keep skip and log a warning instead?
that would be better practice

Reply:
Yep, we can also do that, np

Reply:
done in https://github.com/foundry-rs/foundry/pull/5848/commits/1b135a568f549d6e405eb8f7e3d9812fc36db237